### PR TITLE
MM-41578 - replace duplicated item with the ldap feature description

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
This PR replaces a duplicated feature list item with a new one that describes the LDAP sync feature.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-41578

#### Screenshots
Captura de pantalla 2022-02-09 a las 13 15 10

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by julmondragon — APPROVED** [review]
Looks good!

File: components/admin_console/billing/plan_details/plan_details.test.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            </WrapperComponent>,
+        );
+
+        const wrapperProfessional = shallow(
+            <WrapperComponent>
+                {featureList(CloudProducts.PROFESSIONAL, true)}
+            </WrapperComponent>,
+        );
+
+        const wrappers = [wrapperProfessional, wrapperEnterprise, wrapperStarter];
```
[the remark this answers is not in the document]
Comment (pablovelezvidal) [reply components/admin_console/billing/plan_details/plan_details.test.tsx:77]: makes sense to review that one too! Thanks.

File: components/admin_console/billing/plan_details/plan_details.test.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    test('should match snapshot when running paid tier and starter', () => {
+        const wrapper = shallow(
+            <WrapperComponent>
+                {featureList(CloudProducts.STARTER, true)}
+            </WrapperComponent>,
+        );
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('all feature items must have different values', () => {
```
Comment (JulienTant) [inline components/admin_console/billing/plan_details/plan_details.test.tsx:58]: Very nice test!

**Review by JulienTant — APPROVED** [review]
Good job @pablovelezvidal! LGTM

**Review by stevemudie — APPROVED** [review]
AD/LDAP group sync present as expected. All set to go @pablovelezvidal!
AD:LDAP group sync

**Pull request merged**